src/dali/plugin/input/rest/binance_com.py

```diff
@@ -142,10 +142,17 @@
 _LOCKED_STAKING = "Locked Staking"
 _SOLO_AIRDROP = "SOLO airdrop"
 _GENERAL_STAKING = "STAKING"
-
-_AIRDROP_LIST = [_SOLO_AIRDROP]
-_INTEREST_LIST = [_FLEXIBLE, _FLEXIBLE_SAVINGS, _LOCKED, _LOCKED_SAVINGS]
-_STAKING_LIST = [_ETH_STAKING, _LOCKED_STAKING, _BNB_VAULT, _LAUNCH_POOL, _GENERAL_STAKING]
+# More types added
+_DEFI_STAKING = "Defi Staking"
+_DEFI_STAKING_OLD = "DeFi Staking"
+_CASH_VOUCHER = "Cash Voucher"
+_LAUNCHPAD = "launchpad"
+_SAVINGS_TRAIL_FUND = "Savings Trail Fund"
+_NFT_AIRFROP_NOV_BTTC = "NFT airfrop Nov-BTTC"
```

Comment:
You can do the same thing here. Change [this line](https://github.com/macanudo527/dali-rp2/blob/4969740753cc453810e7d384d8534b4d177f752a/src/dali/plugin/input/rest/binance_com.py#L800) from:
```
if dividend[_EN_INFO] in _AIRDROP_LIST or re.search("[aA]irdrop", dividend[_EN_INFO]):
```
to
```
if dividend[_EN_INFO] in _AIRDROP_LIST or re.search("[aA]ir[df]rop", dividend[_EN_INFO]):
```
And then you can delete this variable here and from the list.